problems/numberOfDigits.js

```diff
@@ -7,8 +7,13 @@
 
 */
 
-function numberOfDigits() {
-
+function numberOfDigits (num, count = 0) {
```

Comment:
I see this code. Nice job finding an answer online, make sure that it makes sense. You can also turn your number into a string and check it's length. This answer is a little bit fancier.